pimd/pim_nht.c

```diff
@@ -643,6 +678,23 @@ static int pim_ecmp_nexthop_search(struct pim_instance *pim,
 			continue;
 		}
 
+		if (!PIM_IF_TEST_PIM(((struct pim_interface *)ifp->info)->options)) {
+			if (PIM_DEBUG_PIM_NHT) {
+				char addr_str[INET_ADDRSTRLEN];
+				pim_inet4_dump("<addr?>", src->u.prefix4,
+						addr_str, sizeof(addr_str));
+				zlog_debug(
+					"%s: pim not enabled on input interface %s(%s) (ifindex=%d, RPF for source %s)",
+					__PRETTY_FUNCTION__, ifp->name,
```

Comment:
`__func__`, but at the moment we are in the progress of removing function names from debug at all https://github.com/FRRouting/frr/pull/10656.

Reply:
Done

Reply:
The PR #10656 is still under discussion, so we can keep the func for now ?

Reply:
OK, then `__func__`

Reply:
Done